routes/movies.routes.js

```diff
@@ -0,0 +1,107 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+
+const Celebrities = require("../models/Celebrity.model");
+const Movies = require("../models/movies.model");
+
+// todas las rutas localhost:3000/movies/.... Y SOLO ESTAS
+const router = require("express").Router();
+
+// all your routes here
+
+router.get("/", (req, res) => {
+    res.render("movies/movies")
+    //res.redirect("/") para volver a inicio
+})
+
+router.get("/add-movies", (req, res) => {
+
+
+    Celebrities.find()
+        .then(Celebridad => res.render("movies/new-movie", { Celebridad }))
```

Comment:
```suggestion
        .then(celebridad => res.render("movies/new-movie", { celebridad }))
```
parametros en minúscula